src/MongoDB/WriteConcern.c

```diff
@@ -344,6 +344,12 @@ static HashTable* php_phongo_write_concern_get_properties_hash(phongo_compat_obj
 
 			if (is_bson) {
 				ZVAL_INT64(&z_wtimeout, wtimeout);
+			} else if (is_serialize) {
+				if (wtimeout > INT32_MAX || wtimeout < INT32_MIN) {
+					ZVAL_INT64_STRING(&z_wtimeout, wtimeout);
+				} else {
+					ZVAL_LONG(&z_wtimeout, wtimeout);
+				}
```

Comment:
I'm not sure if it's already covered in the tests, but I'll double-check that we're testing this code path of encoding `wtimeout` according to its range. I think some of the existing tests might be expecting either a string or integer (with `%r`), which is no longer necessary since the serialization output will now be determined entirely by the value.

Reply:
Confirmed that this _is_ covered by the "64-bit wtimeout is always encoded as as string" test cases in `writeconcern-serialization-001.phpt` and `writeconcern-serialization-002.phpt`.